### ISSUE TITLE
Accessibility: Can't use the editor with a screen reader

### ISSUE THREAD

**Issue by francipvb** [dart-lang/dart-pad#1978]
## What happened?

The issue is that I am a screen reader user and I can't write text into the dartpad editor because it has accessibility issues.

## Steps to reproduce problem

Just enable or start a screen reader and try to edit text in the editor. You'll see that the screen reader cannot follow the insertion point, nor read anything while in focus mode.

## Additional info

The problem is the text editor itself. I found that [Monaco](https://github.com/Microsoft/monaco-editor) editor has a better screen reader support. It is used inside `VScode`.

### Browser

Not browser-related.

### Machine

Tested under windows.

Version:

### Your code

Not relevant.

**Comment by parlough** [comment]
Thanks for opening the issue. I'm going to close this as a duplicate of https://github.com/dart-lang/dart-pad/issues/1072.

Looks like this would involve incorporating a new editor, such as Monaco (like you mentioned) or the 6.x version of Codemirror which is a complete remake.
